app/helpers/hosts_helper.rb

```diff
@@ -135,6 +135,23 @@ def host_global_status_class(status)
     end
   end
 
+  def host_global_status_link_class(status)
+    if status.status_link
```

Comment:
Undefined method `status_link` for integer:

![status-link](https://user-images.githubusercontent.com/2664090/96413518-0d6af700-11ec-11eb-8f62-bd53a575175b.png)

Caused by giving `status.to_global` as an arg when the method is called in host_description_helper.rb#56

Reply:
I fixed this bug it should not happen again.